extensions/github/package.json

```diff
@@ -407,7 +445,7 @@
     "@parcel/watcher": "^2.4.1",
     "@raycast/eslint-config": "^1.0.8",
     "@types/lodash": "^4.17.5",
-    "@types/node": "^20.14.5",
```

Comment:
You should not use 22 since Raycast is always using LTS. The current LTS is 20.

Reply:
Hmm this is just `devDependencies` and I usually do `npm i @types/node@ts5.5 -D` to keep it consistent with the typescript version. In production, raycast will use node@lts (20) anyways. Plus, we have the preference to use production node in development mode. I think this is a non-issue and being used in other commands too: [search](https://github.com/search?q=repo%3Araycast%2Fextensions+%40types%2Fnode%5C%22%3A+%2F%22%5C%5E22%2F&type=code).

Reply:
I don't think so. Some Node.js APIs are different in 22, which will confuse developers.

Reply:
Just curious, what API changes are we concerned about here? [All changes](https://nodejs.org/en/blog/announcements/v22-release-announce) I could see do not impact raycast development (especially when you turn on production mode). I can revert the version, but given v22 will enter LTS in October, I'd like to understand if this is a blocking change? If it is, I can create a tangent PR to move all extensions using v22 back to v20.

Reply:
If this is not blocking, then I would prefer to future-proof with this change, or revert if any other changes are required

Reply:
This is more of your own personal preference, which leaves a potential compatibility issue for the project, and you want others to help you clean up the mess after the problem, which is not appropriate.

This is an open-source project, and we should try to include as few personal preferences as possible without practical significance.

Reply:
I'm with @jfkisafk on this one. Sure, some Node.JS apis are different, but we're not really using them. Only if you are really using Node specific APIs like `fs` it **might** break (although the diffe. That's not the case here...

Reply:
Thanks for raising the bar guys! Based on the discussion, I will resolve this thread as this is a non-blocking issue. As suggested above, if `@types/node@22` shows incompatibility, I will personally revert all extensions back to v20. I am not expecting others to clean it up!